docs/data/joy/components/skeleton/BasicSkeleton.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import AspectRatio from '@mui/joy/AspectRatio';
+import Card from '@mui/joy/Card';
+import Skeleton from '@mui/joy/Skeleton';
+import Typography from '@mui/joy/Typography';
+
+export default function BasicSkeleton() {
+  return (
+    <Card variant="outlined" sx={{ width: 343, display: 'flex', gap: 2 }}>
+      <AspectRatio ratio="21/9">
+        <Skeleton variant="overlay">
+          <img
+            alt=""
```

Comment:
But I don't understand this demo. As far as I understand the use case for skeleton, it's to wait for the API to return the data, so we don't know the img URL at this point, so why is not a Skeleton with not child?

```suggestion
```

If it's because the image already has its layout, then

```suggestion
          <img
            alt=""
            src="data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=" 
```

would make more sense to me